### ISSUE TITLE
Cannot download the jenkins war file and plugins

### ISSUE THREAD

**Issue by sangwook-bak** [jenkins-infra/helpdesk#3661]
### Problem with this page

https://get.jenkins.io/war-stable/2.401.2/jenkins.war

### Expected behavior

The file should be downloaded

### Actual behavior

It says 502 error instead.
Example)
https://get.jenkins.io/war-stable/2.401.2/jenkins.war
https://updates.jenkins.io/download/plugins/role-strategy/665.v4d4f871dc768/role-strategy.hpi

Office firewall manager mentioned that outbound firewall towards Internet does not block the site.
Please check for source IPs that used from our office.
203.241.151.50
203.244.221.1
203.244.221.2
202.20.193.254

**Comment by dduportal** [comment]
Hi @sangwook-bak  thanks for reporting.

We do not have see errors on these services and our monitoring + usages shows no errors as well in the past 24 hours: at first sight it is a problem related to your network.

In order for us to investigate, could you provide us the following elements:

- The page (screenshot is OK) at https://get.jenkins.io/war-stable/2.401.2/jenkins.war?mirrorlist on your webbrowser, which shows which mirror will be selected for download in your case (based on your Geographic IP)
- The result of the following command line (to check which redirect and which error you would have):

```
curl --verbose --location https://get.jenkins.io/war-stable/2.401.2/jenkins.war
```


Thanks

**Comment by sangwook-bak** [comment]
Screenshot - it says 502 error
![image](https://github.com/jenkins-infra/helpdesk/assets/12130076/103cc3d7-c34a-4f39-abfa-939e115d09c8)

**Comment by sangwook-bak** [comment]
curl --verbose --location https://get.jenkins.io/war-stable/2.401.2/jenkins.war
*   Trying 20.7.178.24:443...
* connect to 20.7.178.24 port 443 failed: Timed out
* Failed to connect to get.jenkins.io port 443 after 21089 ms: Couldn't connect to server
* Closing connection 0
curl: (28) Failed to connect to get.jenkins.io port 443 after 21089 ms: Couldn't connect to server

**Comment by dduportal** [comment]
Hi @sangwook-bak , there is definitively a problem in your corporate network:

- The HTTP/502 error shown in your webbrowser is not issued by the Jenkins infrastructure. It looks like there is a proxy between your browser and the Internet and this proxy is unable to access our website `get.jenkins.io`

- It is not a DNS problem: the `curl` command resolves to the correct IPv4 (`20.7.178.24`) associated to `get.jenkins.io`:

  ```bash
  $ dig +short get.jenkins.io
   public.publick8s.jenkins.io.
  20.7.178.24
  ```

There is nothing we can do to help you here as it's clearly a proxy/firewall issue. You have to contact your network administrators and check with them so they allow access to the IP `20.7.178.24` at least (and ideally to all mirrors that could be used).

This problem might have been an indirect consequence of [the outage we had last Friday](https://status.jenkins.io/issues/2023-07-07-maintenance-aks-k8s-1-25/) resulting in changing the public IP of `get.jenkins.io`: if your admin have set up `20.119.232.75` (the former IP since te 12th of June - https://www.jenkins.io/blog/2023/06/22/mirrors-jenkins-new-IP/), they have to update their firewall rules again.

**Comment by sangwook-bak** [comment]
Hi. Seems the firewall manager added rule for the given IPs. Thanks for assistance.
